docs/working-with-projects/working-with-images/create-tag-immutability-rules.md

```diff
@@ -3,46 +3,46 @@ title: Tag Immutability Rules
 weight: 85
 ---
 
-By default, users can repeatedly push an image with the same tag to repositories in Harbor. This causes the previous image to effectively be overwritten with each push, in that the tag now points to a different image and the image that previously used the tag now becomes tagless. This is due to the Docker implementation, that does not enforce the mapping between an image tag and the image digest. This can be undesirable in certain cases, because the tag can no longer be trusted to identify the image version. The sha256 digest remains reliable and always points to the same build, but it is not rendered in a human-readable format.
+By default, users can repeatedly push an artifact with the same tag to repositories in Harbor. This causes the previous artifact to effectively be overwritten with each push, in that the tag now points to a different artifact and the artifact that previously used the tag now becomes tagless. This is due to the Docker distribution implementation, that does not enforce the mapping between an image tag and the image digest. This can be undesirable in certain cases, because the tag can no longer be trusted to identify the image version. The sha256 digest remains reliable and always points to the same build, but it is not rendered in a human-readable format.
```

Comment:
do we still rely on the docker distribution implementation in v2.0? if not, we can just say "This is due to the implementation doesn't enforce the mapping between an image tag and the image digest"

Reply:
updated